client/src/components/users/InvitationListItem.vue

```diff
@@ -20,7 +20,8 @@
       <ion-label
         class="cell invitation-label"
       >
-        <span>{{ timeSince(invitation.date, '--', 'short') }}</span>
+        <span>{{ invitation.createdOn }}</span>
+        <!-- <span>{{ timeSince(invitation.createdOn, '--', 'short') }}</span> -->
```

Comment:
why do we keep this as comment?

Reply:
Fixed. It was because `createdOn` is a string, not a DateTime, but I added deserialization.